python/lsst/ts/standardscripts/maintel/mtmount/park_mount.py

```diff
@@ -0,0 +1,110 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.``
+
+__all__ = ["ParkMount"]
+
+import yaml
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+from lsst.ts.xml.enums import MTMount
+
+
+class ParkMount(salobj.BaseScript):
+    """Park Mount for the MTMount.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    None
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Park Mount for the MTMount.")
+
+        self.mtcs = None
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/maintel/mtmount/park_mount.yaml
+            title: ParkMount v1
+            description: Configuration for ParkMount.
+            type: object
+            properties:
+              position:
+                description: The position to park the MTMount.
+                type: string
+                enum: ["ZENITH", "HORIZON"]
+              ignore:
+                  description: >-
+                    CSCs from the group to ignore in status check. Name must
+                    match those in self.group.components, e.g.; hexapod_1.
+                  type: array
+                  items:
+                    type: string
+            additionalProperties: false
+            required: [position]
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def set_metadata(self, metadata):
+        pass
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        self.config = config
+        self.position = MTMount.ParkPosition[config.position]
+
+        if self.mtcs is None:
+            self.mtcs = MTCS(
+                domain=self.domain,
+                intended_usage=MTCSUsages.Slew | MTCSUsages.StateTransition,
+                log=self.log,
+            )
+            await self.mtcs.start_task
+
+        if hasattr(self.config, "ignore"):
+            for comp in self.config.ignore:
+                if comp not in self.mtcs.components_attr:
+                    self.log.warning(
+                        f"Component {comp} not in CSC Group. "
+                        f"Must be one of {self.mtcs.components_attr}. Ignoring."
+                    )
+                else:
+                    self.log.debug(f"Ignoring component {comp}.")
+                    setattr(self.mtcs.check, comp, False)
+
+    async def run(self):
+        await self.mtcs.assert_all_enabled()
```

Comment:
You are already calling `assert_all_enabled` in the park command no? You might to to either remove it from there or from here.